src/main/java/uk/sky/cqlmigrate/ExecutionInfo.java

```diff
@@ -0,0 +1,29 @@
+package uk.sky.cqlmigrate;
+
+import com.datastax.driver.core.ConsistencyLevel;
+import com.datastax.driver.core.Session;
+
+class ExecutionInfo {
```

Comment:
Yes to the rename.  Regarding your other question, can you please clarify?  You can set a consistency level when you create a session, but that will then apply by default to all writes AND reads just the same.  You can't set separate read and write consistency defaults.